interp/notation.ml

```diff
@@ -225,11 +225,9 @@ let declare_delimiters scope key =
   end;
   try
     let oldscope = String.Map.find key !delimiters_map in
-    if String.equal oldscope scope then ()
-    else begin
-      Flags.if_verbose Feedback.msg_info (str "Hiding binding of key " ++ str key ++ str " to " ++ str oldscope);
```

Comment:
You mean turn this message into a warning?

Reply:
Yes, and also doing the `Undelimit` automatically.

Reply:
Isn't that what is currently done?

Reply:
No, the direction "delimiters -> scope" is updated (in `delimiters_map `table) but the direction "oldscope -> no delimiter" is not (in `scope_map` table).